src/NzbDrone.Core/Tv/MovieService.cs

```diff
@@ -61,6 +62,45 @@ public class MovieService : IMovieService, IHandle<MovieFileAddedEvent>,
             _logger = logger;
         }
 
+
+        public System.Linq.Expressions.Expression<Func<Core.Tv.Movie, bool>> ConstructFilterExpression(string FilterKey, string FilterValue)
+        {
+            if (FilterKey == "all" && FilterValue == "all")
+            {
+                return v => v.Monitored == true || v.Monitored == false;
```

Comment:
i realize, this is returning TRUE all the time, but i wasnt sure if i could just not specify a filter value here... if you know .. let me know.. otherwise i might try later. it would be better to just return true instead of the expression.. though the compiler should optimize it out anyway if it can....